students/ArchWu/Session4/dict_lab.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+def dict1():
+    """
+    Create a dictionary containing “name”, “city”, and “cake” for “Chris” from “Seattle” who likes “Chocolate” (so the keys should be: “name”, etc, and values: “Chris”, etc.)
+    Display the dictionary.
+    Delete the entry for “cake”.
+    Display the dictionary.
+    Add an entry for “fruit” with “Mango” and display the dictionary.
+    Display the dictionary values.
+    Display whether or not “cake” is a key in the dictionary (i.e. False) (now).
+    Display whether or not “Mango” is a value in the dictionary (i.e. True).
+
+    """
+
+    print(record1)
+    for v in record1:
+        if 'cake' in v:
+            v.remove('cake')
+    print(record1)
+    record1['fruit'] = 'Mango'
+    print(record1)
+    print(record1.values())
+    exist_mango = False
+    for v in record1.values():
+        if 'cake' in v:
+            print("Error!")
+        exist_mango = exist_mango or 'Mango' in v
+    print(exist_mango)
+
+def dict2():
+    """ dict tracking number of t's in keys """
+    print(record1)
+    record2 = {}
+    for k, v in record1.items():
+        count = 0
+        for letter in v:
```

Comment:
instead of going through each character at once, you can use the .count() function to count how many t's there are in a string. You may want to lower the string so all t's are lower-case when searching